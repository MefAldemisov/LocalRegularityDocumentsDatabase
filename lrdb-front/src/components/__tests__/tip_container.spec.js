import { shallowMount } from "@vue/test-utils";
import tipContainer from "../tips/tip_container.vue";

describe("TodoList", () => {
    it("should render to a snapshot when tips list has items", () => {
        const propsData = { tips: ["search", "load"] };
        const wrapper = shallowMount(tipContainer, { propsData });
        expect(wrapper.element).toMatchSnapshot();
    });

    it("the order of elements in the props shouldn't affect fie behaviour", () => {
        const propsData = { tips: ["load", "search"] };
        const wrapper = shallowMount(tipContainer, { propsData });
        expect(wrapper.element).toMatchSnapshot();
    });
});
