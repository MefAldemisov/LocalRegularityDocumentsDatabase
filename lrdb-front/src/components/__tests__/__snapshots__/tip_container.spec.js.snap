// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList should render to a snapshot when tips list has items 1`] = `
<aside>
  <load-tip-stub />
   
  <!---->
   
  <!---->
   
  <!---->
</aside>
`;

exports[`TodoList the order of elements in the props shouldn't affect fie behaviour 1`] = `
<aside>
  <load-tip-stub />
   
  <!---->
   
  <!---->
   
  <!---->
</aside>
`;
